documentation/python-sdk-guide/modules/ROOT/pages/index.adoc

```diff
@@ -94,51 +94,65 @@ poetry update
 
 === Loading an Aspect Model
 
+*Note*: the examples below will be using a
+hthttps://github.com/eclipse-esmf/esmf-aspect-model-editor/blob/main/core/apps/ame/src/assets/aspect-models/org.eclipse.examples.movement/1.0.0/Movement.ttl
```

Comment:
* Typo at the beginning of the line (hthttp)
* I'd rather write "The examples below will be using the ..."